packages/web/components/Landing/JoinUs.tsx

```diff
@@ -1,25 +1,57 @@
-import { Box, Button, Container, HStack, Text, VStack } from '@metafam/ds';
-import BackgroundImage from 'assets/landing/sections/section-6.png';
+/* eslint-disable no-nested-ternary */
+import {
+  Box,
+  Button,
+  Container,
+  HStack,
+  Text,
+  useBreakpointValue,
+  VStack,
+} from '@metafam/ds';
+import BackgroundImage from 'assets/landing/sections/section-7.png';
 import { FullPageContainer } from 'components/Container';
 import { StartButton } from 'components/Landing/StartButton';
 import { MetaLink } from 'components/Link';
 import { useOnScreen } from 'lib/hooks/useOnScreen';
-import { useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
 import { FaDiscord, FaGithub, FaTwitter } from 'react-icons/fa';
 
 import { LandingFooter } from './LandingFooter';
+import { Rain } from './OnboardingGame/Rain';
 
 export const JoinUs: React.FC = () => {
   const ref = useRef<HTMLDivElement>(null);
   const onScreen = useOnScreen(ref);
+  const root = typeof window !== 'undefined' ? document.body : null;
+  const [noMotion, setNoMotion] = useState(true);
+  const displayElement = noMotion ? true : !!onScreen;
+  const buttonSize = useBreakpointValue({ base: 'sm', xl: 'lg' });
   const section = 'join-us';
 
+  useEffect(() => {
```

Comment:
this useEffect and its content is being used in a lot of places. Worth wrapping it into a separate hook that can be called in all these places.

We could also store `no-motion` in localStorage. Its my assumption that if someone is opting for "no motion" then it should be persisted the next time they visit our site.

Reply:
> this useEffect and its content is being used in a lot of places. Worth wrapping it into a separate hook that can be called in all these places.
> 
> We could also store `no-motion` in localStorage. Its my assumption that if someone is opting for "no motion" then it should be persisted the next time they visit our site.

I am already getting the users `prefers-reduced-motion` preference on loading the page which sets `MotionPreference` in `localStorage`. The effects toggle also updates the value. I take your point about moving it to a separate hook, so will add that to my list. 👍🏻